Sources/ShopifyCheckoutKit/AnalyticsEvents.swift

```diff
@@ -0,0 +1,4594 @@
+//   let pixelEventsPaymentInfoSubmittedData = try PixelEventsPaymentInfoSubmittedData(json)
+//   let pixelEventsPaymentInfoSubmitted = try PixelEventsPaymentInfoSubmitted(json)
+//   let pixelEventsProductAddedToCartData = try PixelEventsProductAddedToCartData(json)
+//   let pixelEventsProductAddedToCart = try PixelEventsProductAddedToCart(json)
+//   let pixelEventsProductRemovedFromCartData = try PixelEventsProductRemovedFromCartData(json)
+//   let pixelEventsProductRemovedFromCart = try PixelEventsProductRemovedFromCart(json)
+//   let pixelEventsProductVariantViewedData = try PixelEventsProductVariantViewedData(json)
+//   let pixelEventsProductVariantViewed = try PixelEventsProductVariantViewed(json)
+//   let pixelEventsProductViewedData = try PixelEventsProductViewedData(json)
+//   let pixelEventsProductViewed = try PixelEventsProductViewed(json)
+//   let pixelEventsSearchSubmittedData = try PixelEventsSearchSubmittedData(json)
+//   let pixelEventsSearchSubmitted = try PixelEventsSearchSubmitted(json)
+//   let pixelEvents = try PixelEvents(json)
+//   let attribute = try Attribute(json)
+//   let browserCookie = try BrowserCookie(json)
+//   let browserLocalStorage = try BrowserLocalStorage(json)
+//   let browserSessionStorage = try BrowserSessionStorage(json)
+//   let browser = try Browser(json)
+//   let cart = try Cart(json)
+//   let cartCost = try CartCost(json)
+//   let cartLine = try CartLine(json)
+//   let cartLineCost = try CartLineCost(json)
+//   let checkout = try Checkout(json)
+//   let checkoutLineItem = try CheckoutLineItem(json)
+//   let clientID = try ClientID(json)
+//   let collection = try Collection(json)
+//   let context = try Context(json)
+//   let customData = try CustomData(json)
+//   let customEvent = try CustomEvent(json)
+//   let customer = try Customer(json)
+//   let purpleData = try PurpleData(json)
+//   let discountAllocation = try DiscountAllocation(json)
+//   let discountApplication = try DiscountApplication(json)
+//   let iD = try ID(json)
+//   let image = try Image(json)
+//   let initData = try InitData(json)
+//   let location = try Location(json)
+//   let mailingAddress = try MailingAddress(json)
+//   let moneyV2 = try MoneyV2(json)
+//   let name = try Name(json)
+//   let order = try Order(json)
+//   let pricingPercentageValue = try PricingPercentageValue(json)
+//   let product = try Product(json)
+//   let productVariant = try ProductVariant(json)
+//   let screen = try Screen(json)
+//   let searchResult = try SearchResult(json)
+//   let shippingRate = try ShippingRate(json)
+//   let timestamp = try Timestamp(json)
+//   let transaction = try Transaction(json)
+//   let webPixelsDocument = try WebPixelsDocument(json)
+//   let webPixelsNavigator = try WebPixelsNavigator(json)
+//   let webPixelsWindow = try WebPixelsWindow(json)
+
+import Foundation
+
+// MARK: - PixelEvents
+struct PixelEvents: Codable {
```

Comment:
Is it worth adding a test around deserializing JSON into one or two events.

Maybe also with missing fields, to double check optionality.